distribution/docs/src/main/resources/content/_configuring/reusing-configurations.adoc

```diff
@@ -17,7 +17,7 @@ To export the current configuration settings:
 To import previously exported configuration settings:
 
 . Install ${branding}
-. Restore all external files, softlinks, and directories that would not have been exported and for which warnings would have been generated during export. This could include among other things: external certificates, content store, and monitored directories.
+. Restore all external files, softlinks, and directories that would not have been exported and for which warnings would have been generated during export. This could include among other things such as external certificates or monitored directories.
 . Launch the newly installed ${branding}.
 . Step through the installation process.
 . Make sure to re-enable <<_automatic_start_on_system_boot,automatic startup and shutdown>> of ${branding} on the new system if it was enabled on the originating system.
```

Comment:
🖊 Back-ticks around `migration:import` below.

Reply:
@lessarderic I was asked to remove them

Reply:
@ricklarsen ☝️